algrtm/src/components/PianoRollPlay.js

```diff
@@ -3,8 +3,8 @@ import VideoComponent from "./VideoComponent";
 
 const PianoRollPlay = () => {
   return (
-    <section className="flex flex-col items-center gap-[5.875rem] my-[7.19rem]">
-      <div className="text-neutralLight-neutral10 h5_500 px-[4rem] flex items-center max-w-[51.875rem]">
+    <section className="flex flex-col items-center gap-[2.75rem] md:gap-[5.88rem] my-[4rem] md:my-[7.19rem] w-full">
+      <div className="text-neutralLight-neutral10 h5 md:h5_500 md:px-[7.75rem] xl:px-[4rem] flex items-center max-w-[51.875rem] w-full">
```

Comment:
In Figma, this element uses `1024/H5 SB` typography for resolutions up to 1024, and after 1440 uses `1440/H5 SB` (mainly it is a difference in font size)